src/components/navigation/www/accountnav.jsx

```diff
@@ -0,0 +1,101 @@
+const classNames = require('classnames');
+const FormattedMessage = require('react-intl').FormattedMessage;
+const injectIntl = require('react-intl').injectIntl;
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const Avatar = require('../../avatar/avatar.jsx');
+const Dropdown = require('../../dropdown/dropdown.jsx');
+
+require('./navigation.scss');
+
+const AccountNav = ({
+    classroomId,
+    isEducator,
+    isOpen,
+    isStudent,
+    profileUrl,
+    thumbnailUrl,
+    username,
+    onClick,
+    onClickLogout,
+    onClose
+}) => (
+    <React.Fragment>
```

Comment:
I think it would make sense for this to include the wrapping component. What is the reason for leaving it out?

Reply:
Sorry, I don't understand. Which wrapping component? NavigationBox?

Reply:
I mean that this is returning a `Fragment` rather than a self-contained component. Where it's pulled from is a `li` (but sometimes we make that element type configurable with an `as` prop for stuff like this).

Reply:
Hm, I think we need to talk offline, I have several questions